package.json

```diff
@@ -15,7 +15,9 @@
     "lint:md-fix": "wp-scripts lint-md-docs --ignore LICENSE.md --ignore node_modules --ignore vendor --fix",
     "lint:style": "wp-scripts lint-style 'resources/**/*.scss'",
     "lint:style-fix": "wp-scripts lint-style 'resources/**/*.scss' --fix",
-    "start": "wp-scripts start"
+    "start": "wp-scripts start",
+    "prebuild": "if test \"$CI\" = \"true\" ; then composer install --no-dev ; fi",
+    "prebuild:dev": "if test \"$CI\" = \"true\" ; then composer install --no-dev ; fi"
```

Comment:
Those two are the same. Am I missing something?

Reply:
the first one runs before `yarn build`, the second one runs before `yarn build:dev`